internal/docker/runner.go

```diff
@@ -2,27 +2,32 @@ package docker
 
 import (
 	"context"
+	"errors"
 	"fmt"
-	"github.com/rs/zerolog/log"
-	"github.com/saucelabs/saucectl/cli/runner"
-	"github.com/saucelabs/saucectl/cli/streams"
-	"github.com/saucelabs/saucectl/internal/fleet"
-	"github.com/saucelabs/saucectl/internal/yaml"
 	"io"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 	"time"
 
+	"github.com/rs/zerolog/log"
+
 	"github.com/saucelabs/saucectl/cli/command"
 	"github.com/saucelabs/saucectl/cli/config"
 	"github.com/saucelabs/saucectl/cli/progress"
+	"github.com/saucelabs/saucectl/cli/runner"
+	"github.com/saucelabs/saucectl/cli/streams"
+	"github.com/saucelabs/saucectl/internal/fleet"
+	"github.com/saucelabs/saucectl/internal/yaml"
 )
 
 // DefaultProjectPath represents the default project path. Test files will be located here.
 const DefaultProjectPath = "/home/seluser"
 
+// ErrEmptyImageProperty represents error regarding empty image base property in the config file.
+var ErrEmptyImageProperty = errors.New("Image base property was not specified in your config file")
```

Comment:
DItto.
```suggestion
var ErrEmptyImageProperty = errors.New("image base property was not specified in your config file")
```

Reply:
I removed this error and left one common for the empty docker image.